client/payment-details/dispute-details/dispute-summary-row.tsx

```diff
@@ -0,0 +1,131 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { dateI18n } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { HorizontalList } from 'wcpay/components/horizontal-list';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { reasons } from 'wcpay/disputes/strings';
+import { formatStringValue } from 'wcpay/utils';
+import { isAwaitingResponse } from 'wcpay/disputes/utils';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+import Paragraphs from 'wcpay/components/paragraphs';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+
+interface Props {
+	dispute: Dispute;
+	daysRemaining: number;
+}
+
+const DisputeSummaryRow: React.FC< Props > = ( { dispute, daysRemaining } ) => {
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	const disputeReason = formatStringValue(
+		reasons[ dispute.reason ]?.display || dispute.reason
+	);
+	const disputeReasonSummary = reasons[ dispute.reason ]?.summary || [];
+
+	const columns = [
+		{
+			title: __( 'Dispute Amount', 'woocommerce-payments' ),
+			content: formatCurrency( dispute.amount, dispute.currency ),
```

Comment:
The issue and mockups specify that this should be in the store's currency. Currently, I have this is set to the dispute currency. 

IMO we should keep these amounts in the same currency and I believe changing this headline amount is out of scope.

![image](https://github.com/Automattic/woocommerce-payments/assets/57298/a1d7a5cb-b482-4b1e-9753-5a8535e8193e)

Reply:
> IMO we should keep these amounts in the same currency and I believe changing this headline amount is out of scope.

I was looking at updating this as part of #6974 – I agree though, I'll open a separate issue for the headline amount.

Reply:
Issue #7124 opened and slack discussion about scope here: p1693885409513359-slack-C03LA109BKM